mim/commands/download.py

```diff
@@ -35,10 +35,17 @@
     cls=OptionEatAll,
     required=True,
     help='Config ids to download, such as resnet18_8xb16_cifar10')
+@click.option(
+    '--no-check-certificate',
+    'check_certificate',
+    is_flag=True,
+    default=True,
+    help='Ignore ssl certificate check')
 @click.option(
     '--dest', 'dest_root', type=str, help='Destination of saving checkpoints.')
 def cli(package: str,
         configs: List[str],
+        check_certificate: bool = True,
```

Comment:
Adding a new parameter in the middle position may cause a bc issue. Suggest moving it to the end.